docs/source/model_design/vivarium_model_components/risk_factors/risk_mediation/index.rst

```diff
@@ -30,4 +30,399 @@ Risk factor mediation
 .. contents::
   :local:
 
-:download:`Please see this power point overview <Risk-Risk Correlation.pptx>`.
+What is mediation?
+------------------
+
+Risk factor mediation is when one risk factor affects an outcome through another risk factor.
+
+For example, in this causal diagram, the risk factor R1 has an effect on the outcome O,
+but that effect is (completely) mediated by another risk factor R2.
+
+.. image:: mediation.drawio.svg
+
+.. note::
+  This diagram is so simple it may not always be described as "mediation."
+  For example, on our :ref:`Causal Diagrams page <general_dags>` page we call this an
+  :ref:`"intermediate." <general_dags_intermediates>`
+  However, it can be useful to analyze this situation as mediation; for example, this
+  causal structure of GBD risks would report a mediation factor
+  (see GBD section for more on this).
+
+However, a diagram involving mediation can be more complex:
+a risk effect may be mediated through multiple (chains of) mediating risk factors,
+and there may be a non-mediated effect as well.
+
+.. image:: mediation-complex.drawio.svg
+
+At the top of this diagram, we see multi-layer mediation through the chain
+of R1→R2→R3→O, where R1-R4 are different risks and O is an outcome.
+Starting from the end of the chain:
+
+* R3 has a direct effect on O
+* R2 has an effect on O mediated through R3
+* R1 has an effect on O mediated through R2 **and** R3
+
+Additionally, R1 has another effect on O mediated through R4,
+and the bottom arrow in the diagram represents another, direct (non-mediated) causal
+effect.
```

Comment:
I think we should be comprehensive and list all of the causal pathways between the risks and O, including the direct effects of R1 and R4.